### PR TITLE
Fixed appearances of content tabs

### LINKED ISSUE #1788: Wrong tabs when changing from internal link to content
image

### PR DESCRIPTION
**Pull request by danrot**
__tasks:__

- [ ] test coverage
- [ ] gather feedback for my changes
- [ ] submit changes to the documentation

__informations:__

| q                | a
| ---------------- | ---
| Fixed tickets    | fixes #1788
| BC breaks        | none
| Documentation PR | none

### CODE REVIEW

File: src/Sulu/Bundle/ContentBundle/Resources/public/js/components/content/main.js [hunk 1]
```diff
@@ -936,17 +916,45 @@ define([
...
+
+            if (this.data.nodeType === constants.internalLinkNodeType
+                || this.data.nodeType === constants.externalLinkNodeType
+            ) {
+                usedTabs = ['seo'];
+            } else if (!!this.data.shadowOn) {
+                usedTabs = [];
+            }
+
+            unusedTabs = _.difference(tabs, usedTabs);
```
Comment (wachterjohannes) [inline src/Sulu/Bundle/ContentBundle/Resources/public/js/components/content/main.js]: nice :+1:

File: src/Sulu/Bundle/ContentBundle/Resources/public/js/components/content/main.js [hunk 2]
```diff
@@ -21,23 +21,21 @@ define([
 
         SHOW_GHOST_PAGES_KEY = 'column-navigation-show-ghost-pages',
 
-        /**
-         * node type constant for content
-         * @type {number}
-         */
-        TYPE_CONTENT = 1,
```
Comment (wachterjohannes) [inline src/Sulu/Bundle/ContentBundle/Resources/public/js/components/content/main.js]: this variable is used a few times in this file which results in that you cannot open a page form any more (:

Comment (danrot) [inline src/Sulu/Bundle/ContentBundle/Resources/public/js/components/content/main.js]: Actually I thought I have replaced all occurences... Will check again.

Comment (danrot) [inline src/Sulu/Bundle/ContentBundle/Resources/public/js/components/content/main.js]: Ok, missed two of them, should be fine now.

**Comment by wachterjohannes** [comment]
its not related to this PR but i will write it down here:

* External link without target is possible to save
* Internal link shows a warning be the message is strange formated

image

**Comment by danrot** [comment]
@wachterjohannes But nobody will read that in here :smiley: Can you please create a separate issue? (if there isn't already one)

**Comment by wachterjohannes** [comment]
i am starting currently fixing that (:

**Comment by wachterjohannes** [comment]
https://github.com/sulu-io/sulu/pull/1805

**Pull request merged**